inference-engine/thirdparty/clDNN/src/program.cpp

```diff
@@ -1386,3 +1392,44 @@ void program::set_layout_optimizer_attributes(layout_optimizer& lo) {
         lo.set_optimization_attribute(layout_optimizer::optimization_attributes_type::use_onednn_impls, 1);
 #endif
 }
+
+std::pair<int64_t, int64_t> program::get_estimated_device_mem_usage() {
```

Comment:
Please add unit tests for this functionality with result value checks

Reply:
Added a unittest